### ISSUE TITLE
Requesting new data processing action term

### ISSUE THREAD

**Issue by mobiusklein** [HUPO-PSI/psi-ms-CV#42]
A new term to describe ion mobility/drift time deconvolution
```
[Term]
id: MS:XXXXXX
name: ion mobility deconvolution
def: "The separation of m/z over ion mobility signal into discrete bins or features." [PSI:MS]
is_a: MS:1000543 ! data processing action
```
It is arguable whether this term should derive from `peak picking` since some approaches involve 2D centroiding m/z by drift time signal.

Several tools exist to do this, including Waters and Bruker software that exports to mzML already. In Waters's case, this is combined with peak picking, charge state deconvolution and deisotoping. In Bruker's, only peak picking is done.

**Comment by sneumann** [comment]
Hi, seems that was not yet added. So this is a term under `data processing action`:
https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMS_1000543 .
Where would it be used in the PSI standards and what information / implications could software derive ?
In case of binning, it could inform that a `<spectrum>` contains a specific (range of) ion mobility. In the case of mzTab, you would add this term to `MTD`, and software could expect/look for CCS in the feature table. So the term could be split into one for binning and one relating to feature detection/peak picking. Yours, Steffen

**Comment by mobiusklein** [comment]
I was thinking primarily of mzML when I opened this issue, where this term or a derived term would be used in `<dataProcessing>` elements to denote that the ion mobility dimension was not a continuum.

I agree there should be a term for binning and a term for feature extraction, since one implies isotopic pattern awareness and the other doesn't, but they both discretize the IMS dimension of a spectrum, so their having a shared ancestor to query for in mzML is still useful.

I don't know enough about how mzTab would distinguish between the two configurations. My intuition is that there is an argument to have both terms just in case an intermediary mzTab file recording binning is used to feed a program that will use that to generate a IM feature extracted file, but this is pretty fiddly.

**Comment by mobiusklein** [comment]
Closed by #87
